garage.com/src/cssXpath/xpathproject.txt

```diff
@@ -0,0 +1,31 @@
+1.Page: /text-box. Write the xpath that will return the ‘Full Name’textbox using the ID attribute. Hint: This element has the ‘input’tag.
+
+//input[@id='userName']
+
+2.Page: /buttons. Write the xpath that will return all elements on the page with the ‘button’tag.
+
+//button[@id='doubleClickBtn']
+//button[@id='rightClickBtn']
+//button[@id='zsYrW']
+Or
+//button[@class='btn btn-primary']
+
+3.Page: /webtables. Write the xpath that will return the div with text of ‘Vega’. Ensure to narrow the scope of your xpath to the ReactTableandensure that your xpath is limited to returningcells only. Cells in this table have the class ‘rt-td’. Hint: You must use “Vega”in your xpath.
+
+//div[@class='rt-td'and text()='Vega']
+
+4.Page: /radio-button.Write the xpath that will return the parent element for the radio button with id=’impressiveRadio’.
+
+//input[@id='impressiveRadio']

```

Comment:
Get the parent of this element
//input[@id="impressiveRadio"]/div::parent
or
//input[@id="impressiveRadio"]/div/..